GenericTest.py

```diff
@@ -265,8 +265,10 @@ def validate_schema(self, payload, schema):
         checker = jsonschema.FormatChecker(["ipv4", "ipv6", "uri"])
         return jsonschema.validate(payload, schema, format_checker=checker)
 
-    def do_request(self, method, url, data=None):
-        return TestHelper.do_request(method, url, data)
+    def do_request(self, method, url, json=None, data=None, headers=None, auth=None, params=None):
```

Comment:
Would this work as something like `def request(self, method, url, **kwargs):`? Just thinking it may avoid having to update the method signature in two places if we just keep the authoritative version in TestHelper.